book/03-hacking-atom/1-hacking-atom.asc

```diff
@@ -27,6 +27,8 @@ include::sections/A03-textmate.asc[]
 
 === Summary
 
-( List of topics we covered / appendix? )
+If you finished this chapter, you should be an Atom-hacking master. We've discussed how you should work with CoffeeScript, and how to put it to good use in creating packages. You should also be able to do this in your own created theme now.
```

Comment:
Actually, I've been reviewing the docs quite a bit over the past couple days and this appears to be the same style as the others :grinning: